xrpl/asyncio/transaction/main.py

```diff
@@ -297,3 +300,9 @@ async def _calculate_fee_per_transaction_type(
 
     # Round Up base_fee and return it as a String
     return str(math.ceil(base_fee))
+
+
+async def _fetch_account_delete_fee(client: Client) -> int:
+    server_state = await client.request_impl(ServerState())
+    fee = server_state.result["state"]["validated_ledger"]["reserve_inc"]
```

Comment:
I think this should be `reserve_base` instead of `reserve_inc` for the account delete fee

https://xrpl.org/server_state.html#server_state

Reply:
No, it's `reserve_inc` - `reserve_base` is the account creation fee/minimum account balance for an existing account. 
```
"validated_ledger": {
        "base_fee": 10,
        "close_time": 683153081,
        "hash": "B52AC3876412A152FE9C0442801E685D148D05448D0238587DBA256330A98FD3",
        "reserve_base": 20000000,
        "reserve_inc": 5000000,
        "seq": 65887201
      },
```
If you look at this example, you can see that `reserve_base` is 20 XRP and `reserve_inc` is 5 XRP. The account delete fee was 5 XRP, not 20.

Reply:
Ah I mis-counted the Zeros - I thought it was 2XRP/5XRP but that's not the case.